dashboards-observability/public/framework/redux/store/index.ts

```diff
@@ -5,21 +5,45 @@
 
 import { configureStore } from '@reduxjs/toolkit';
 import rootReducer from '../reducers';
+import storage from 'redux-persist/lib/storage';
```

Comment:
Have we considered using url parameters? That way users can share a page with given filters or bookmark a page with commonly used filters for convenience?

Reply:
Yes, this is a feature, you may create an issue for a feature request.